### ISSUE TITLE
Search field should be focused automatically when opening the add rooms to space dialog

### ISSUE THREAD

**Issue by anoadragon453** [vector-im/element-web#17063]
![image](https://user-images.githubusercontent.com/1342360/116068045-ad18ae80-a681-11eb-9c53-121c4c1b39bb.png)

It would be handy to focus the search field immediately when adding rooms to a space, so that users don't need to perform an extra click before they start searching.
